packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -32,12 +31,11 @@ const CHAT_CLASS = 'dx-chat';
 const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 
 type Properties = ChatProperties & {
-  title: string;
+  dayHeaderFormat?: Format;
```

Comment:
I don't understand why this change is here.

Reply:
https://github.com/DevExpress/DevExtreme/pull/28327/commits/1ab518ce0c6da4dbd1ea075181642bb35f0dcb41